### PR TITLE
Add class with tables

### PR DESCRIPTION
**Pull request by jindrichjehlicka**

### CODE REVIEW

File: server/src/models/desk.model.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  averageWorkhoursUsage10days: number;
-  averageDailyUsage10days: number;
-  shortUsagesCount10days: number;
+  averageWorkHoursUsage: number;
+  averageDailyUsage: number;
+  shortUsagesCount: number;
 };
 
-export type DeskStatus = "Active" | "Inactive" | "Archived";
+export type DeskStatus = "active" | "inactive" | "archived";
```
Comment (Zejnilovic) [inline server/src/models/desk.model.ts]:
```suggestion
export type DeskStatus = "active" | "inactive" | "offline";
```

File: server/src/controllers/desks.controllers.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
-import { RequestHandler } from "express";
-import { getDesksSer } from "../services/desks.service";
+import { RequestHandler } from 'express';
+import { getDesksService } from '../services/desks.service';
 
+type RequestParams = {
+  tableIds?: string[]
```
Comment (MatyasVondra) [inline server/src/controllers/desks.controllers.ts]: deskIds pls

File: server/src/services/desks.service.ts [hunk 3]
```diff
@@ -1,31 +1,37 @@
-import { Desk } from "../models/desk.model";
+import { DeskDto } from '../models/desk.model';
 
-export function getDesksSer(): Desk[] {
-  console.log("Fetching desks");
+
+type DesksServiceParams = {
+  tableIds?: string[]
```
Comment (MatyasVondra) [inline server/src/services/desks.service.ts]: deskIds pls

File: server/src/models/desk.model.ts (hunk 1 above)
Comment (jindrichjehlicka) [reply server/src/models/desk.model.ts]: done

File: server/src/controllers/desks.controllers.ts (hunk 2 above)
Comment (jindrichjehlicka) [reply server/src/controllers/desks.controllers.ts]: done

File: server/src/services/desks.service.ts (hunk 3 above)
Comment (jindrichjehlicka) [reply server/src/services/desks.service.ts]: done

File: server/src/controllers/desks.controllers.ts (hunk 2 above)
Comment (MatyasVondra) [reply server/src/controllers/desks.controllers.ts]: Pushnuls to?

**Review by Zejnilovic — DISMISSED** [review]

**Review by MatyasVondra — APPROVED** [review]

**Pull request merged**